### DISCUSSION TITLE
Searching doesn't include the description field?

### DISCUSSION THREAD

**Discussion by jbwhit** [massCodeIO/massCode#273] [Q&A]
I was looking to see if I should port my collection of txt files to massCode, and in testing it out, I found that the descriptions weren't searched. I'll give you an example, I wrote

title: "Pandas timestamp string"
description: "This is how to format a timestamp string"
and the code snippet body:

```python
pd.Timestamp('now').strftime("%Y%m%d%H%M%S%f")
```

I would have expected if I were to search "format" it would have found that snippet, but no. Is this the intended behavior?

**Comment by antonreshetov** [comment]
Yes, this is intended behavior. The search is performed only by snippet name or by its body.
